test/unit/csv_file_from_public_host_test.rb

```diff
@@ -11,73 +11,114 @@ def stub_csv_request(status: 206, body: '', path: 'some-path')
       .to_return(status: status, body: body)
   end
 
-  test '#new yields a temporary file' do
+  test '.new yields a temporary file' do
     stub_csv_request
 
-    CsvFileFromPublicHost.new('some-path') do |file|
+    response = CsvFileFromPublicHost.csv_response('some-path')
+    CsvFileFromPublicHost.new(response) do |file|
       assert File.exist?(file.path)
     end
   end
 
-  test '#new handles a very long filename' do
```

Comment:
I thought you were going to keep this test, @chrislo. Did you change your mind?

Irrespective of whether we keep the test or not, I think we should update the commit note (titled "Avoid dependency on attachment path for tempfile prefix") to explain that using a string, instead of truncating the attachment path, for the temporary filename also fixes the problem that was previously fixed in 18bc4340b7ce0a2fd13a48261f6575a77e11119a.

Reply:
Yes, I decided to remove it as it wasn't adding much value after the change to how we generate tempfile names, and it avoided having to change it to keep it passing with the changes to how stubs work. I'll make a note of that in the commit note.